docs/02-getting-started/02-installation.md

```diff
@@ -88,7 +88,10 @@ are looking for Linux support.
 The Wing Console is a desktop application that allows you to interact with your
 Wing applications running locally on the cloud simulator.
 
-1. Download the latest version of the **Wing Console** for [macOS](https://wing-console.s3.amazonaws.com/wing-console.dmg), [macOS arm64](https://wing-console.s3.amazonaws.com/wing-console-arm64.dmg) or [windows](https://wing-console.s3.amazonaws.com/wing-console.exe)
+1. Download the latest version of the **Wing Console** for your operating system:
```

Comment:
```suggestion
1. Download the latest version of the **Wing Console**:
```

I like the change to show this as a `ul`, but I think saying "operating system" here is redundant (and technically kinda wrong, since macOS and macOS arm64 are not _really_ different OSes)